feature_engine/timeseries/forecasting/base_forecast_transformers.py

```diff
@@ -61,10 +66,17 @@ class BaseForecastTransformer(BaseEstimator, TransformerMixin, GetFeatureNamesOu
     def __init__(
         self,
         variables: Union[None, int, str, List[Union[str, int]]] = None,
+        fill_value: Union[float, int, str] = None,
         missing_values: str = "raise",
         drop_original: bool = False,
     ) -> None:
 
+        if not isinstance(fill_value, (float, int, str)):
```

Comment:
This is a param for pandas shift, so I would leave the checks to pandas. I would remove this from here. Not sure how many options this parameter can take, and also they may expand it or reduce it in the future. So no checks makes maintaining easier.